api/speeches/post.ts

```diff
@@ -192,8 +193,12 @@ const getApi = async ({
   )
     .toString("utf8")
     .split(":");
+  console.log(`authEmail`, authEmail);
+  console.log(`authToken`, authToken);
+  console.log(`user.rawToken`, user.rawToken);
   if (user.rawToken !== authToken || email !== authEmail) return undefined;
   const password = AES.decrypt(inputPassword, user.otter.key).toString(encutf8);
+  console.log(`password`, password);
```

Comment:
Dont log password. Do a .length, or if you must, a slice of the last 4 characters